src/CotMG/Helper.tsx

```diff
@@ -6,11 +7,21 @@ import { StaneksGift } from "./StaneksGift";
 export let staneksGift = new StaneksGift();
 
 export function loadStaneksGift(saveString: string): void {
-  if (saveString) {
-    staneksGift = JSON.parse(saveString, Reviver) as StaneksGift;
-  } else {
+  let staneksGiftData: unknown;
+  try {
+    staneksGiftData = JSON.parse(saveString, Reviver);
+  } catch (error) {
+    console.error(error);
```

Comment:
Can we merge this condition into the other error handling? If someone edits their file incompetently, it would be good to be notified (more than just on the console)